content/pages/publishAlgo.json

```diff
@@ -0,0 +1,69 @@
+{
+  "title": "Publish",
+  "description": "Highlight the important features of your algorith to make it more discoverable and catch the interest of data consumers.",
```

Comment:
might be cleaner to split this up into fixed page content and forms content since this is content which should not change on tab change, and we can minimize maintenance duplication. So a `pages/publish.json` where we only have the page title, description, and warning (warning should go above tabs anyway).

Then either put both forms with different keys in this file (like `formDataset` & `formAlgorithm`) or create like so we end up with files:

- `pages/publish.json`
- `pages/publish-forms.json`

Or it might be already time to create a whole content folder for the publish page since this is constantly extended, where we could use:

- `pages/publish/index.json`
- `pages/publish/forms.json`, or `pages/publish/form-dataset.json` & `pages/publish/form-algorithm.json`

Then the page component just queries for the index file content, and each form can query the exact content they need in their components instead of passing all content as prop.

Reply:
Yes good point, also thought about that, I'll do it as soon as the Custom Image selection is ready.